imgui-sdl2-support/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "imgui-sdl2-support"
+version = "0.8.1-alpha.0"
+edition = "2018"
+authors = ["The imgui-rs Developers"]
+description = "sdl2 support code for the imgui crate"
+homepage = "https://github.com/imgui-rs/imgui-rs"
+repository = "https://github.com/imgui-rs/imgui-rs"
+license = "MIT/Apache-2.0"
+categories = ["gui"]
+
+[dependencies]
+imgui = { version = "0.8.1-alpha.0", path = "../imgui" }
+sdl2 = "^0.34.5"
+
+[dev-dependencies]
+glow = "0.10.0"
+imgui-glow-renderer = { version = "0.8.1-alpha.0", path = "../imgui-glow-renderer" }
+sdl2 = { version = "^0.34.5", features = ["bundled", "static-link"] }
```

Comment:
I'll take a more thorough look later (at work now) but we shouldn't be turning either of these features on for people (bundled is particualrly bad because it has UB on 64-bit windows ([for example](https://github.com/Rust-SDL2/rust-sdl2/blob/master/sdl2-sys/sdl_bindings.rs#L951) long is 32 bit on win64 but size_t is 64 bit)

Also `^` is the default so you don't need to specify it.

Reply:
It's just for the examples

Reply:
Ah, right, i missed this was under dev-dependencies.

Reply:
didn't know about that UB though!